R/api_cache.R

```diff
@@ -0,0 +1,98 @@
+#' @title Data Caching API
+#' @description Functions to manage cache data across sessions.
+#' @param path character, path to cache directory, Default: cache_dir()
```

Comment:
```suggestion
#' @param path character, path to cache directory, Default: `cache_dir()`
```
would recommend to be consistent with this, so that you don't have to amend the wordlist for spelling with the code stuff